Tests/SwiftDocCTests/Semantics/SymbolTests.swift

```diff
@@ -492,7 +492,7 @@ class SymbolTests: XCTestCase {
         XCTAssertEqual(withRedirectInArticle.redirects, nil)
 
         XCTAssertEqual(problems.first?.diagnostic.identifier, "org.swift.docc.UnsupportedDocCommentDirective")
-        XCTAssertEqual(problems.first?.diagnostic.range?.lowerBound.line, 3)
+        XCTAssertEqual(problems.first?.diagnostic.range?.lowerBound.line, 15)
```

Comment:
This test seems like it's finding another directive in some other symbol. Please update the test to 
- don't build more content that it needs to
- warn about some unsupported directive that's created in this test.

Reply:
It's referring to the `@Redirected` directive in the test, but I can see how that's confusing. I had made `makeDocumentationNodeForSymbol` add an arbitrary offset of 12 to the doc comment's line numbers, but I changed it so that it takes the value as a parameter instead for the specific tests I wrote.